exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricConfiguration.java

```diff
@@ -55,10 +55,20 @@
    */
   public abstract Duration getDeadline();
 
+  /**
+   * Returns the strategy for how to send metric descriptors to Cloud Monitoring.
+   * 
+   * <p>The Default is to only send descriptors once per process/classloader.
```

Comment:
Do we need to say a bit more about the "per classloader" possibility? E.g., what's the right way to set up the classpath under OSGi or an app server, and what behavior to expect?

(This depends on our audience, which I don't understand very well at the moment.)

Reply:
Likely, yes.   It's a really nuanced/hard question that's MOST relevant to auto-instrumentation.  I'm not sure what I'd add here yet, but I'll noodle on it.